jcli/src/jcli_app/certificate/mod.rs

```diff
@@ -98,6 +98,10 @@ pub enum Error {
     VoteEncryptingKey,
     #[error("invalid bech32 public key, expected {expected} hrp got {actual}")]
     InvalidBech32Key { expected: String, actual: String },
+    #[error("Invalid shares JSON representation")]
+    InvalidJson(#[from] serde_json::Error),
+    #[error("Invalid binary share")]
```

Comment:
This should be clearer worded, I think: "invalid binary share data"?
Capitalization should be more consistent across the enum. Let's go with small letters everywhere.

Reply:
my bad should have realized it :/